.env

```diff
@@ -19,5 +19,6 @@ PUBLISHER_BASE_URL=
 REFRESH_ACCESS_TOKEN_ENDPOINT=null
 SEGMENT_KEY=null
 SITE_NAME=null
+SUPPORT_EMAIL=null
```

Comment:
Will this populate from frontend-platform for prod env?

Reply:
I don't think so. I think there are corresponding edx-platform configs but they won't automatically apply here. This will need soem changes.

Reply:
Do you know why all these variables are set to null for prod env?

Reply:
I think these were all set to `null` in [frontned-template-application](https://github.com/edx/frontend-template-application/blob/master/.env), (they are now blank strings).